src/linear_algebra/basic.lean

```diff
@@ -161,6 +161,13 @@ instance unique_of_left [subsingleton M] : unique (M →ₗ[R] M₂) :=
 instance unique_of_right [subsingleton M₂] : unique (M →ₗ[R] M₂) :=
 coe_injective.unique
 
+lemma eq_zero_iff (f : M →ₗ[R] M₂) : f = 0 ↔ ∀ m, f m = 0 :=
+begin
```

Comment:
I think `linear_map.ext_iff` might be a proof of this lemma

Reply:
Basically yes.

In fact a proof is `simp only [ext_iff, zero_apply]` which convinces me that this lemma shouldn't even exist so I'll drop it.